<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-V3TS40K" failures="1" tests="5" name="Test" time="65.712" errors="0" timestamp="2022-05-23T09:48:10 CEST">
  <testcase classname="tests.ValidateNavBarTest" name="titleAssert" time="0.238"/>
  <testcase classname="tests.LoginTest" name="login" time="1.452"/>
  <testcase classname="tests.ValidateTitleTest" name="titleAssert" time="0.196"/>
  <testcase classname="tests.ForgetPasswordTest" name="resetPass" time="1.859"/>
  <testcase classname="tests.LoginTest" name="login" time="1.97">
    <failure type="java.lang.AssertionError" message="expected [aidah1803@gmail.com] but found [komora.aida@gmail.com]">
      <![CDATA[java.lang.AssertionError: expected [aidah1803@gmail.com] but found [komora.aida@gmail.com]
at tests.LoginTest.login(LoginTest.java:45)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- login -->
</testsuite> <!-- Test -->
